src/locale/la.js

```diff
@@ -0,0 +1,71 @@
+// Latin [la]
+import dayjs from 'dayjs'
+
+const monthFormat = 'ianuarii_februarii_martii_aprilis_maii_iunii_iulii_augusti_septembris_octobris_novembris_decembris'.split('_')
+const monthStandalone = 'ianuarius_februarius_martius_aprilis_maius_iunius_iulius_augustus_september_october_november_december'.split('_')
+const MONTHS_IN_FORMAT = /D[oD.ᵒ]?(\[[^[\]]*\]|\s)+MMMM?|MMMM?(\[[^[\]]*\]|\s)+D[oD.ᵒ]?/
+const months = (dayjsInstance, format) => {
+  if (MONTHS_IN_FORMAT.test(format)) {
+    return monthFormat[dayjsInstance.month()]
+  }
+  return monthStandalone[dayjsInstance.month()]
+}
+months.s = monthStandalone
+months.f = monthFormat
+
+// src: https://stackoverflow.com/a/9083076
+function romanise(num) {
+  const key = [
+    '', 'C', 'CC', 'CCC', 'CD', 'D', 'DC', 'DCC', 'DCCC', 'CM',
+    '', 'X', 'XX', 'XXX', 'XL', 'L', 'LX', 'LXX', 'LXXX', 'XC',
+    '', 'I', 'II', 'III', 'IV', 'V', 'VI', 'VII', 'VIII', 'IX'
+  ]
+  const digits = String(+num).split('')
+  let roman = ''
+  let i = 3
+  while (i > 0) {
+    i -= 1
+    roman = (key[+digits.pop() + (i * 10)] || '') + roman
+  }
+  return Array(+digits.join('') + 1).join('M') + roman
+}
+
+const locale = {
+  name: 'la',
+  weekdays: 'Dominica_feria secunda_feria tertia_feria quarta_feria quinta_feria sexta_Sabbato'.split('_'),
+  weekdaysShort: 'Dominica_feria II_feria III_feria IV_feria V_feria VI_Sabbato'.split('_'),
+  weekdaysMin: 'Dom._II_III_IV_V_VI_Sab.'.split('_'),
+  months,
+  monthsShort: 'ian._feb._mar._apr._mai._iun._iul._aug._sep._oct._nov._dec.'.split('_'),
+  weekStart: 0,
+  yearStart: 4,
+  ordinal: n => [romanise(n), 'ᵒ'].join(''),
+  // The relative time variables are in nominative case only
+  relativeTime: {
+    future: 'ad %s',
```

Comment:
You can see the output in the [locale test (L64-80)](https://github.com/iamkun/dayjs/blob/049bf1dbb2a5d210d078135510d2f19d80f25807/test/locale/la.test.js#L64-L80), in the `T` constant (it is a multi-dimensional array consisting of `[number, 'English word', 'number Latin word']`; I think it is quite readable :wink:).

Reply:
No, it isn't readable. It's code, DRY as a desert.

Reply:
Okay, here’s a simple ‘conversion‘ table:

| English | Latin |
|---|---|
| a few seconds | paucæ secundæ |
| a minute | minuta |
| 2 minutes | 2 minutæ |
| an hour | hora |
| 3 hours | 3 horæ |
| a day | dies |
| 3 days | 3 dies |
| a month | mensis |
| 2 months | 2 menses |
| a year | annus |
| 2 years | 2 anni |

Reply:
Just pinging you, @igneus. When you find some time, please check the table above if it is sufficiently readable for you. All other issues you have raise were fixed in the latest commit.

Reply:
I'm not sure I'll manage to communicate what I really wanted to see, so I cloned your repo and checked out the `latin` branch with hope that I will be able to play with the code myself, but it turns out that my NodeJS skills don't suffice to load the package from development sources.

I ran `npm run-script build`, managed to load `dayjs.min.js` in the interactive console (`var dayjs = require('dayjs.min.js');`), but attempts to load locales fail, probably due to my lack of understanding of some basic NodeJS concepts.

![2020-10-21-201523_1366x768_scrot](https://user-images.githubusercontent.com/53101/96765852-89c32d00-13da-11eb-8e9f-93d230bf4ebd.png)

Since I personally don't care much about dayjs' Latin locale, I probably just give up and :see_no_evil:

Reply:
> I'm not sure I'll manage to communicate what I really wanted to see

I’m sorry I didn’t understand you. Anyway, I still think that you proofed everything in the Latin locale except for those string in the ‘conversion’ table.

Anyway, in the [Czech locale](https://github.com/iamkun/dayjs/blob/dev/src/locale/cs.js#L8:L49), they use different inflection of those strings when they are used in the past (e.g. `a few seconds ago`) or future tense (`in a few seconds`). If any of the strings from `relativeTime` should be in different grammar case in the past and future tenses, we would need to update it accordingly.

> I cloned your repo and checked out the `latin` branch with hope that I will be able to play with the code myself, but it turns out that my NodeJS skills don't suffice to load the package from development sources.

All you should need to do after cloning the repo is as follows:

```bash
# Install the dependencies
npm install

# Build `dayjs`
# Note: This is same as running `npm run-script build`
npm run build

# Run the `la.test.js` test
# Note: Run the following command from repo root or modify both relative paths
node node_modules/jest/bin/jest.js test/locale/la.test.js --coverage=false
```

I think the easiest way to test things without any deeper knowledge of Node.js, it to modify the `test/locale/la.test.js` to include `console.log()` command(s), where you put the output of whatever you want to see.

Reply:
> Since I personally don't care much about dayjs' Latin locale, I probably just give up and see_no_evil

Pinging @igneus. Are you determined to give up on this review? As I understand it, you have not reviewed only one part, the `relativeTime`.

Reply:
Ping, @igneus. No answer from you for more than a month. I presume you are occupied with some other stuff, however, I’d be very happy if you at least say something. :wink: Thanks in advance with all your help! :smiley: